conformance_tests/sysman/test_sysman_device/src/test_sysman_device_helper.cpp

```diff
@@ -74,22 +106,40 @@ int main(int argc, char **argv) {
   auto sysman_devices = lzt::get_zes_devices();
   EXPECT_FALSE(sysman_devices.empty());
 
-  std::vector<zes_uuid_t> sysman_device_uuids;
-  get_sysman_devices_uuids(sysman_devices, sysman_device_uuids);
+  if (strcmp(device_hierarchy, "FLAT") != 0) { // composite or combined mode
+    std::vector<UUID> sysman_device_uuids;
```

Comment:
Can we define these outside of if loop ?

Reply:
For composite/combined & flat case, i have named the core UUID vector's different as per the context. Hence i kept inside the if condition.

Reply:
Updated as per discussion